src/databox/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'databox' Extension
+==========================================
+
+This package is for the 'databox' extension.
+i.e. 'az databox'
```

Comment:
Please add more details on how to use this extension. You can refer to [storage-preview](https://github.com/Azure/azure-cli-extensions/tree/master/src/storage-preview).

Reply:
Thanks, I'll add it.

---
In reply to: [388083315](https://github.com/Azure/azure-cli-extensions/pull/1344#discussion_r388083315) [](ancestors = 388083315)